Makefile

```diff
@@ -151,7 +151,7 @@ dhall_types: ocaml_checks
 
 replayer: ocaml_checks
 	$(info Starting Build)
-	ulimit -s 65532 && (ulimit -n 10240 || true) && dune build src/app/replayer/replayer.exe --profile=testnet_postake_medium_curves
+	ulimit -s 65532 && (ulimit -n 10240 || true) && dune build src/app/replayer/replayer.exe --profile=devnet
```

Comment:
We've used a hard-coded profile here for a long time. I changed the profile, because the ledger depth has changed.

I'll keep an eye out for any issues that arise.